content/en/storage-providers/seal-workers/post-workers.md

```diff
@@ -0,0 +1,155 @@
+
+A lotus worker instance can only be one of the following:
+- WindowPoSt worker
+- WinningPoSt worker
+- Worker for sealing tasks
+
+## Launching PoSt workers
+
+Before launching your PoSt worker you need to ensure that your worker meets the minimal requirements for the job it is going to perfom. You should also consider how the PoSt workers can access the sealed sectors<> and setting the correct enviroment variables before you run the workers.
+
+### Minminal spec requirements
+
+These are the minimal requirements for running each of the PoSt tasks:
+
+| **Task**    | **GPU**            | **VRAM** | **RAM** |
+|:-----------:|:------------------:|:--------:|:-------:|
+| winningPoSt | Highly recommended |          |         |
+| windowPoSt  | Highly recommended | 8\.5GiB  | 128GiB  |
+
+{{< alert icon="callout" >}}
+Although both tasks can be run with a powerful CPU, it is highly recommended to run on GPU as it speeds up the process significantly, and therefore reducing the chance to miss windowPoSt or winningPoSt
+{{< /alert >}}
+
+### Remote storage access
+
+The windowPoSt process requires reading random leafs of all the sealed sectors in a proving deadline. When setting up windowPoSt workers one needs to consider how the workers can access those files. The PoSt workers can ask any other worker to read challenges for them, including the lotus-miner process, but it will prefer reading it from local paths.
+
+{{< alert icon="warning" >}}
+Storage providers should design their worker sectors access according to their setup size and also redundancy required. If only one worker has read access to the sealed sectors, it can create a single point of failure. If you have multiple partitions in a deadline, having multiple PoSt workers requesting a single worker to read challenges for them may cause unwanted I/O load.
+{{< /alert >}}
+
+### Environment variables
+
+Remember to have the appropriate Nvidia-drivers and nvidia-opencl-icd installed if running OpenCL on your worker. If using CUDA, install the CUDA-toolkit and build Lotus with FFI_USE_CUDA=1
+
+Ensure that workers have access to the following environment variables when they run. These are similar to those used by the Miner daemon ([explained in the setup guide]({{< relref "configure#setup" >}})):
+
+```
+export MINER_API_INFO:<TOKEN>:/ip4/<miner_api_address>/tcp/<port>/http
+export BELLMAN_CUSTOM_GPU="MODEL-NAME:CORES" # If you´re using a custom GPU
+export FIL_PROOFS_PARAMETER_CACHE=/fast/disk/folder # > 100GiB!
+export FIL_PROOFS_PARENT_CACHE=/fast/disk/folder2   # > 50GiB!
+```
+
+For the PoSt worker to start, it will need to read and verify the Filecoin proof parameters. These are the same parameters as is currently on the lotus-miner instance. We recomend copying them over from your lotus-miner machine, else they will be downloaded on first run.
```

Comment:
```suggestion
When the PoSt worker starts, it needs to read and verify the Filecoin proof parameters. We recommend copying them over from your lotus-miner machine. Otherwise, they will be downloaded locally on first run.
```